reference/3.0/Microsoft.PowerShell.Management/Start-Service.md

```diff
@@ -226,39 +243,49 @@ Aliases: wi
 
 Required: False
 Position: Named
-Default value: None
+Default value: False
 Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
 ### CommonParameters
-This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see about_CommonParameters (http://go.microsoft.com/fwlink/?LinkID=113216).
+
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable,
+-InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose,
+-WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
+
 ## INPUTS
 
 ### System.ServiceProcess.ServiceController, System.String
-You can pipe objects that represent the services or strings that contain the service names to Start-Service.
-## OUTPUTS
 
-### None or System.ServiceProcess.ServiceController
-When you use the PassThru parameter, Start-Service generates a System.ServiceProcess.ServiceController object representing the service.
-Otherwise, this cmdlet does not generate any output.
-## NOTES
-* You can also refer to Start-Service by its built-in alias, "sasv". For more information, see about_Aliases.
+You can pipe objects that represent the services or strings that contain the service names to this
+cmdlet.
 
-  Start-Service can control services only when the current user has permission to do so.
-If a command does not work correctly, you might not have the required permissions.
+## OUTPUTS
 
-  To find the service names and display names of the services on your system, type "get-service".
-The service names appear in the Name column, and the display names appear in the DisplayName column.
+### None, System.ServiceProcess.ServiceController
 
-  You can start only the services that have a start type of "Manual" or "Automatic".
-You cannot start the services with a start type of "Disabled".
-If a Start-Service command fails with the message "Cannot start service \<service-name\> on computer," use a Get-WmiObject command to find the start type of the service and, if necessary, use a Set-Service command to change the start type of the service.
+This cmdlet generates a **System.ServiceProcess.ServiceController** object that represents the
+service, if you specify **PassThru**. Otherwise, this cmdlet does not generate any output.
+
+## NOTES
 
-  Some services, such as Performance Logs and Alerts (SysmonLog) stop automatically if they have no work to do.
-When Windows PowerShell starts a service that stops itself almost immediately, it displays the following message: "Service \<display-name\> start failed."
+* You can also refer to `Start-Service` by its built-in alias, **sasv**. For more information, see
```

Comment:
Since an alias is a cmdlet, should it be codefenced instead of bold?

Reply:
fixed